expo/scheduler/functions.py

```diff
@@ -391,6 +398,14 @@ def day_to_cal_time(day='Saturday', week=datetime(2015, 02, 19, tzinfo=pytz.time
     return cal_times
 
 
+def cal_times_for_conf(conference):
+    from gbe.functions import get_conference_days
+    days = get_conference_days(conference)
+    first_day = days[0].day
```

Comment:
Does this assume that the first day in the get_conference_days call is the first day of the conference?  If so, then the get_conference_days should force a sort order.  It currently does not.